fearless/Common/Helpers/AccountRepositoryFactory.swift

```diff
@@ -7,15 +7,22 @@ protocol AccountRepositoryFactoryProtocol {
 
     func createAccountRepository(for networkType: SNAddressType)
         -> AnyDataProviderRepository<AccountItem>
+    func createStreambleProvider(for accountAddress: AccountAddress) -> StreamableProvider<AccountItem>
 }
 
 final class AccountRepositoryFactory: AccountRepositoryFactoryProtocol {
     let storageFacade: StorageFacadeProtocol
     let operationManager: OperationManagerProtocol
```

Comment:
Looks like we are not using it. Let's remove?

Reply:
operationManager used at line 70
```swift
return StreamableProvider<AccountItem>(
            source: AnyStreamableSource(EmptyStreamableSource()),
            repository: AnyDataProviderRepository(repository),
            observable: AnyDataProviderRepositoryObservable(observable),
            operationManager: operationManager
        )
```